UEBPrep/spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
I will look into writing tests this week. Wasn't familiar enough with RSpec to dive into it with this sprint.